client-java/controller/src/main/java/org/evomaster/client/java/controller/db/SqlScriptRunner.java

```diff
@@ -145,16 +145,30 @@ public static void runCommands(Connection connection, List<String> commands) {
     }
 
 
-    public static void execInsert(Connection conn, List<InsertionDto> insertions) throws SQLException {
+    //From DTO Insertion Id to generated Id in database
+    public static Map<Long, Long> map = new HashMap<>();
```

Comment:
first, mutable fields should not be `public`
second, I am usually against static mutable fields (apart from very special cases). Read what I wrote in the `for_developers.md` file. I think this should be refactored. If not possible, we need to discuss it